README.md

```diff
@@ -9,121 +9,34 @@ to events and their promotion.
 
 ## Installation
 
-mobilizon-reshare is distributed through Pypi. Use 
+`mobilizon_reshare` is distributed through Pypi and DockerHub. Use 
 
-```pip install mobilizon-reshare```
+```shell
+$ sudo pip install mobilizon-reshare
+```
 
 to install the tool in your system or virtualenv.
 
 This should install the command `mobilizon-reshare` in your system. Use it to access the CLI and discover the available
 commands and their description.
 
-## Scheduling and temporal logic
+We know that using `pip` to interact with your system interpreter is not the best practice in the world, but right now we don't
```

Comment:
Is this really necessary? I would move this to some comment if we really feel we want to apologize. Documentation should be much more dry and contain minimal information.
If you want to reference the Guix Package, just do that in a dedicated section.

Reply:
We can definitely rephrase it but promoting `pip` as the default way to distribute mobilizon reshare is going to open the way to many angry users in github issues i believe.

First of all when you pip install something that depends on a package already installed in your system - it may be a dependency of anything you installed with `apt` (or `rpm` or whatever), think of `requests` - pip will need to resolve the conflict and it won't always be possible. Also pip is not great for [general purpose package managing](https://stackoverflow.com/questions/7915998/does-uninstalling-a-package-with-pip-also-remove-the-dependent-packages).

There's no easy solution to this right now, besides the docker image but that's also not great for regular CLI usage. I think the optimal way to deliver this software to (Linux) users is distros packages, right now we can't afford to maintain a package for every distribution out there. It would be wonderful if someone volunteered or if we manage to engage someone from the internet.

Reply:
I totally agree on the technical considerations. I still believe an excusation non petita is a bit out of place in the documentation